File: python questions/new_year.py
for _ in range(int(input())):
    n = int(input())

    rem = n/2020
    if (rem < 1):
        print('NO')
    else:
        lastdigit = n%2020
        if (lastdigit > rem):
            print("NO")
        else:
            print('YES')

File: c++ questions/newYear.cpp
#include<iostream>
using namespace std;

int main(){
    int t;
    cin>>t;

    for(int i = 0;i<t;i++){
        int n;
        cin>>n;

        int rem = n/2020;
        if (rem<1){
            cout<<"NO"<<endl;
        } else{
            int lastdigit = n%2020;
            if (lastdigit > rem){
                cout<<"NO"<<endl;
            }
            else{
                cout<<"YES"<<endl;
            }
        }
    }
    return 0;
}
